### PR TITLE
Show warning messages in dialogs

### PR DESCRIPTION
**Pull request by geffrak**
DESCRIPTION

Visualy notify the user instead of printing warning messages in the log file.

### CODE REVIEW

File: hooks/submitter_create.py [hunk 1]
```diff
@@ -40,7 +41,10 @@ def can_submit(self):
         """
 
         if not self.__create_client_module.is_create_installed():
-            self.log_warning("Shotgun Create is not installed.")
+            QtGui.QMessageBox.warning(
+                None, "Cannot submit to Shotgun", "Shotgun Create is not installed!",
```
Comment (jfboismenu) [inline hooks/submitter_create.py]: Could we provide a hyperlink to where to download in the box?

File: hooks/submitter_sgtk.py [hunk 2]
```diff
@@ -43,10 +39,11 @@ def can_submit(self):
         """
 
         if not self._upload_to_shotgun and not self._store_on_disk:
-            self.log_warning(
-                "App is not configured to store images on disk nor upload to shotgun!"
+            QtGui.QMessageBox.warning(
+                None,
+                "Cannot submit to Shotgun",
+                "App is not configured to store images on disk nor upload to shotgun!",
```
Comment (jfboismenu) [inline hooks/submitter_sgtk.py]:
```suggestion
                "Application is not configured to store images on disk or upload to shotgun!",
```

File: hooks/submitter_create.py (hunk 1 above)
Comment (geffrak) [reply hooks/submitter_create.py]: The design is to open the browser. The next call open your browser on the right page.

Comment (geffrak) [reply hooks/submitter_create.py]:
```
self.__create_client_module.open_shotgun_create_download_page(
                 self.__app.sgtk.shotgun		                 self.__app.sgtk.shotgun
             )
)
```

Comment (jfboismenu) [reply hooks/submitter_create.py]: Uh, what, how did I miss this! Sorry!

**Pull request merged**